packages/typst.node/src/lib.rs

```diff
@@ -134,11 +134,23 @@ pub struct QueryDocArgs {
     pub field: Option<String>,
 }
 
+#[napi(string_enum)]
+#[derive(serde::Serialize, serde::Deserialize, Debug, PartialEq, Eq)]
+pub enum NodePdfStandard {
```

Comment:
This enum seems too complicated. We can simply pass string option instead. From typst's definition, we can deserialize a pdf standard by `let standard: PdfStandard = serde_json::from_value(standard_option)`.

https://github.com/typst/typst/blob/5c37a1cfea2a08b08f4af34ba867748fd90c687e/crates/typst-pdf/src/lib.rs#L140